framework/contrib/hit/lex.h

```diff
@@ -154,6 +154,10 @@ class Lexer
   std::vector<Token> _tokens;
 };
 
+/// lex is *the* function in the hit namespace. It takes a given hit input text and returns a list
+/// of tokens. This is/was helpful for syntax highlighting in the peacock editor
+std::vector<Token> tokenize(const std::string & fname, const std::string & input);
```

Comment:
This doc comment needs to be updated/fixed.  The name of the function is `tokenize` not `lex` - which is what the (copy pasted) doc comment implies.